lib/elaborate/elaborateExpressionInstantiations.ts

```diff
@@ -0,0 +1,52 @@
+import * as O from "../parser/objects";
+
+export function elaborateExpressionInstantiations(file: O.OFile) {
+  for (const obj of file.objectList) {
+    if (obj instanceof O.OName) {
+      if (obj instanceof O.OInstantiation === false && obj.parent instanceof O.OInstantiation === false
```

Comment:
`obj instanceof O.OInstantiation === false` should always be true?

Reply:
no, the concurrent and procedure instantiations get parsed as Oinstantiations directly?